test/modules/TestResultCoordinator/Services/PartitionReceiverHandler.cs

```diff
@@ -62,10 +62,8 @@ public async Task ProcessEventsAsync(IEnumerable<EventData> events)
                         {
                             if (long.TryParse(sequenceNumberFromEvent.ToString(), out long sequenceNumber))
                             {
-                                DateTime enqueuedtime = GetEnqueuedTime(deviceIdFromEvent.ToString(), moduleIdFromEvent.ToString(), eventData);
-
                                 // TODO: remove hardcoded eventHub string in next line
-                                var testResult = new MessageTestResult((string)moduleIdFromEvent + ".eventHub", enqueuedtime)
+                                var testResult = new MessageTestResult((string)moduleIdFromEvent + ".eventHub", DateTime.UtcNow)
```

Comment:
Why did this change to UtcNow?

Reply:
Now, we will use our own timestamps in the TRC rather than using EventHub's timestamps. We weren't really doing anything with the eventhub timestamps, and they don't really tell us much. 
I think it's better to use our own timestamps to signal when the TRC actually receives the message (not when eventHub receives the message). It also helps with the logic I'm implementing here. Now we can say "have we received a message from eventhub within the last <EventHubDelayTolerance>," whereas otherwise that is more difficult to answer